layouts/partials/jslibraries.html

```diff
@@ -183,3 +183,41 @@
 
   amplitude.getInstance().logEvent("Page loaded");
 </script>
+
+// OS detection script
+<script>
+  function detectOS() {
+    const userAgent = window.navigator.userAgent.toLowerCase();
+    if (userAgent.indexOf("win") > -1) return "windows";
+    if (userAgent.indexOf("mac") > -1) return "mac";
+    if (userAgent.indexOf("linux") > -1) return "linux";
+    return "unknown";
+}
+
+function updateOSInfo() {
+    const os = detectOS();
+    console.log(os);
+
+    // All possible IDs for each OS
+    const osElements = {
+        windows: ["windows", "windows-tab", "tab-windows"],
+        mac: ["mac", "mac-tab", "tab-mac"],
+        linux: ["linux", "linux-tab", "tab-linux"]
+    };
+
+    for (const [currentOS, ids] of Object.entries(osElements)) {
+        ids.forEach(id => {
```

Comment:
Do not do it like this. Change the content to use some uniform ID in both app and cli pages.

then use bootstrap js to change the tab. 

eg.
```js
let windowsTab = document.getElementById('windows-tab');
let tab = new bootstrap.Tab(windowsTab);
tab.show()

```